### ISSUE TITLE
Incompatible with Rubidium.

### ISSUE THREAD

**Issue by Romoslayer** [Harleyoc1/ChunkAnimator#24]
Rubidium is a forge port of the sodium mod and Rubidium overhauls Chunk Animators chunk rendering animation and you get the fade in chunks similar to vanilla mc, was hoping there was a way to fix this or make Chunk Animator work with rubidium.

Rubidium: 1.16.5-0.2.5 - https://www.curseforge.com/minecraft/mc-mods/rubidium
Chunk Animator: 1.16.5-1.2.4

**Comment by papertek** [comment]
will this be fixed?

**Comment by MstDominik** [comment]
Crash with rubidium 1.19.2(0.6.1) :'(

**Comment by ChrisClickbait** [comment]
Still an issue. Also on 1.18.2

**Comment by zachsai** [comment]
Still an issue
[latest.log](https://github.com/Harleyoc1/ChunkAnimator/files/10144234/latest.log)

**Comment by HeuristikV64** [comment]
Still an issue

**Comment by GeminiMarshdevil** [comment]
still an issue, 
on 1.18.2
forge: 40.2.1

**Comment by KateHanami** [comment]
still happening on 1.19.2... optifine is growing unstable and relying on it is not doing any good

**Comment by Umbreeunix** [comment]
Still an issue.

**Comment by DanMcgraw** [comment]
had to go through 140 mods to discover this, is infact, still an issue 1.19.2

**Comment by Darkmega18** [comment]
this may also be my issue too. have a crash log.

[crash-2023-06-11_19.14.45-client.txt](https://github.com/Harleyoc1/ChunkAnimator/files/11714268/crash-2023-06-11_19.14.45-client.txt)

**Comment by MadAnomaly** [comment]
Still an issue on 1.20.1

[latest.log](https://github.com/Harleyoc1/ChunkAnimator/files/11915897/latest.log)

**Comment by sandalle** [comment]
MC: 1.20.1
Forge: 1.20.1-47.0.35
Chunk Animator: 1.3.7
Rubidium (fork of Sodium modified for Forge): 0.6.5

[crash-2023-07-02_01.04.09-client.txt](https://github.com/Harleyoc1/ChunkAnimator/files/11928960/crash-2023-07-02_01.04.09-client.txt)

**Comment by Stoutscientist** [comment]
stiiiiiiiiiillllllllllll an issue

**Comment by cobaltlol** [comment]
Crashes on 1.18.2

**Comment by dracoattila** [comment]
Still crashing on 1.20.1

Rubidium 0.6.5 - Chunk Animator 1.20.1-1.3.7

**Comment by Stoutscientist** [comment]
I don't think this is going to get patched

**Comment by cobaltlol** [comment]
fr

**Comment by calreveraster** [comment]
Also experiencing this for 1.16.5 :( Hoping to see a fix for it soon

**Comment by Mqony** [comment]
Any updates on this or is the mod deprecated?

**Comment by xFirefalconx** [comment]
1.18.2 Modpack, added this mod with both versions, each crash.
[crash-2023-10-11_17.08.51-client.txt](https://github.com/Harleyoc1/ChunkAnimator/files/12872829/crash-2023-10-11_17.08.51-client.txt)
[crash-2023-10-11_17.15.37-client.txt](https://github.com/Harleyoc1/ChunkAnimator/files/12872830/crash-2023-10-11_17.15.37-client.txt)
[latest.log](https://github.com/Harleyoc1/ChunkAnimator/files/12872837/latest.log)

Doesn't matter which version. 
Oculus/Rubidium here too.

**Comment by wickedits** [comment]
at this point someone should make a new chunk animator, compatible with rubidium

**Comment by undefined06855** [comment]
yeah, this is an issue on I think all versions, at least it is on 1.16.5

**Comment by rafaelkai** [comment]
Still an issue.

**Comment by foragedysetour** [comment]
Still an issue.

**Comment by Minebrah** [comment]
still an issue

**Comment by Ghost-Ship-Supreme** [comment]
Still an issue... (1.18.2)

**Comment by NovaViper** [comment]
I want to add that Embeddium (the fork of Rubidium) is also incompatible on 1.21 (and pretty much the entirety of 1.20.x), it crashes with "Mixin apply for mod chunkanimator failed".

Full log: https://mclo.gs/7344t18

**Comment by flleeppyy** [comment]
sill an issue on 1.19.2

**Comment by xGabou** [comment]
still an issue on 1.20.1

**Comment by Cicopath** [comment]
Still an issue on 1.20.1: https://mclo.gs/uhU0mII
Linking [this Embeddium issue ticket here](https://github.com/FiniteReality/embeddium/issues/11) since it's relevant.

**Comment by amzelee** [comment]
+1 to the people having the issue. Anyways I don't think the author has any intention of fixing this.

**Comment by SxilverKat** [comment]
Added support for this in my mod:

https://www.curseforge.com/minecraft/mc-mods/chunk-animator-embeddium-compat
